data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,8 @@
+export default function countValues(stack) {
+  let answer = 0;
+  while (stack.peek() !== undefined) {
+    stack.pop();
```

Comment:
You could just use `pop` in the `while` expression and remove it from here. Then it would just be `pop` instead of `peek`/`pop`.